queenbee/cli/assets/new-recipe.yaml

```diff
@@ -0,0 +1,34 @@
+metadata:
+  version: 0.1.0
+dependencies:
+- type: operator
+  name: whalesay
+  version: 0.1.0
+  source: https://pollination.github.io/shed # Replace with name of repo
```

Comment:
so this can be a local folder or a repository?

Reply:
It's pretty buggy if it's a local folder so I would only use this option for testing (avoids us mocking the repository)